docs/guide/setup-examples/julia.md

```diff
@@ -0,0 +1,401 @@
+---
+title: How to use Sublime for Julia development
+---
+
+Adapted from https://github.com/PetrKryslUCSD/HowToUseJuliaWithSublimeText3
+This guide was written by a Windows user,
+that uses the Windows Subsystem for Linux to run Julia.
+
+When working with Julia,
+I like having my source code on the left
```

Comment:
I'm unsure how to handle the usage of "I" in a documentation like this, because the author is unknown for the reader, until they check who contributed to this page in the source code. I would prefer either passive wording or an explicit mention of who the author is.